sdcm/utils/common.py

```diff
@@ -1444,7 +1444,7 @@ def get_scylla_gce_images_versions(project: str = SCYLLA_GCE_IMAGES_PROJECT, ver
         filters = "(family eq 'scylla(-enterprise)?')(name ne .+-build-.+)"
 
         if version and version != "all":
-            filters += f"(name eq '.*scylla(-enterprise)?-{version.replace('.', '-')}.*')"
+            filters += f"(name eq 'scylla(db)?(-enterprise)?-{version.replace('.', '-')}.[0-9](-rc[0-9])?')"
```

Comment:
i'm not sure it is enough, as if it will be `2022.1.11`, then 1 digit won't be enough...
i have tried, and it is still missing something:

```
$ hydra list-images -c gce -r europe-west1 -v 2021.1
+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
|                                                                        GCE Machine Images by version                                                                         |
+---------+----------------------------+-------------------------------------------------------------------------------------------------------+-------------------------------+
| Backend | Name                       | ImageId                                                                                               | CreationDate                  |
+---------+----------------------------+-------------------------------------------------------------------------------------------------------+-------------------------------+
| GCE     | scylla-enterprise-2021-1-9 | https://www.googleapis.com/compute/v1/projects/scylla-images/global/images/scylla-enterprise-2021-1-9 | 2022-02-24T04:22:47.286-08:00 |
| GCE     | scylla-enterprise-2021-1-8 | https://www.googleapis.com/compute/v1/projects/scylla-images/global/images/scylla-enterprise-2021-1-8 | 2022-01-12T11:40:24.867-08:00 |
| GCE     | scylla-enterprise-2021-1-7 | https://www.googleapis.com/compute/v1/projects/scylla-images/global/images/scylla-enterprise-2021-1-7 | 2021-11-24T12:20:16.657-08:00 |
| GCE     | scylla-enterprise-2021-1-6 | https://www.googleapis.com/compute/v1/projects/scylla-images/global/images/scylla-enterprise-2021-1-6 | 2021-10-20T07:46:20.512-07:00 |
| GCE     | scylla-enterprise-2021-1-5 | https://www.googleapis.com/compute/v1/projects/scylla-images/global/images/scylla-enterprise-2021-1-5 | 2021-08-25T05:03:58.653-07:00 |
| GCE     | scylla-enterprise-2021-1-4 | https://www.googleapis.com/compute/v1/projects/scylla-images/global/images/scylla-enterprise-2021-1-4 | 2021-07-29T05:32:51.337-07:00 |
| GCE     | scylla-enterprise-2021-1-3 | https://www.googleapis.com/compute/v1/projects/scylla-images/global/images/scylla-enterprise-2021-1-3 | 2021-07-19T04:46:11.766-07:00 |
| GCE     | scylla-enterprise-2021-1-2 | https://www.googleapis.com/compute/v1/projects/scylla-images/global/images/scylla-enterprise-2021-1-2 | 2021-06-23T10:17:56.437-07:00 |
+---------+----------------------------+-------------------------------------------------------------------------------------------------------+-------------------------------+
```

meaning, it is missing `2021.1.10` and forward...